index.html

```diff
@@ -98,6 +98,7 @@ <h1 class="hero-title">&nbsp;OPENCODE</h1>
  <hr class="card-underline">
  <div class="row quote">
    <p class="hero-quote">IIIT Allahabad's very own mini GSOC!</p>
+   <a href="https://www.facebook.com/opencodeiiita/" class="btn btn-primary" style="background-color: red;">Like Us on Facebook</a>
```

Comment:
This is not what we want. Read more about adding like button on facebook docs.

Reply:
I didn't get what to do then. I simply added a button to like facebook page of Opencode18. where is the docs ?

Reply:
Plz explain what to do?

Reply:
Read here https://developers.facebook.com/docs/plugins/like-button

Reply:
I have updated code now, is this fine or needs more improvement